client/src/Pages/Profile/UserInfo/UserAvatar/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import Axios from 'axios';
+import { Typography } from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+
+import Button from '../../../../Components/Button';
+import useStyles from './style';
+
+export default function index() {
```

Comment:
Instead of `index` you should give this the component name `UserAvatar` - it'll help with debugging